packages/babel-preset-env/test/normalize-options.spec.js

```diff
@@ -87,6 +87,18 @@ describe("normalize-options", () => {
       });
     });
 
+    it("should throw removed option if corejs version is 2", () => {
+      [2, 2.1].forEach(corejs => {
+        ["entry", "usage"].forEach(useBuiltIns => {
+          expect(() =>
+            normalizeOptions.default({ useBuiltIns, corejs }),
+          ).toThrowError(
+            /Since Babel 8, the core-js@2 support has been dropped. Please use `corejs: "3.6"`/,
```

Comment:
Can we also mention that if they _really_ want `core-js@2`, they can use `babel-plugin-polyfill-corejs2` from https://github.com/babel/babel-polyfills?